Index: app/src/main/java/com/example/boardgameapp/splashscreen.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.boardgameapp\r\n\r\nimport android.content.Intent\r\nimport androidx.appcompat.app.AppCompatActivity\r\nimport android.os.Bundle\r\nimport android.os.Handler\r\n\r\nclass splashscreen : AppCompatActivity() {\r\n    override fun onCreate(savedInstanceState: Bundle?) {\r\n        super.onCreate(savedInstanceState)\r\n        setContentView(R.layout.activity_splashscreen)\r\n\r\n        Handler().postDelayed({\r\n            val intent = Intent(this, MainActivity::class.java)\r\n            startActivity(intent)\r\n            finish()\r\n        }, 3000)\r\n\r\n\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/java/com/example/boardgameapp/splashscreen.kt b/app/src/main/java/com/example/boardgameapp/splashscreen.kt
--- a/app/src/main/java/com/example/boardgameapp/splashscreen.kt	(revision 131ad363c1104482e34f823b0682525b2aec51a0)
+++ b/app/src/main/java/com/example/boardgameapp/splashscreen.kt	(date 1665001129455)
@@ -14,7 +14,7 @@
             val intent = Intent(this, MainActivity::class.java)
             startActivity(intent)
             finish()
-        }, 3000)
+        }, 0)
 
 
     }
